apps/console/src/features/applications/components/settings/sign-on-methods/sign-in-method-customization.tsx

```diff
@@ -558,6 +573,51 @@ export const SignInMethodCustomization: FunctionComponent<SignInMethodCustomizat
         );
     };
 
+    const SMSOTPConflictMessage = () => {
+
+        const FIRST_ENTRY: number = 0;
+        const { idpList } = smsValidationResult;
+        const moreThan1IdP: boolean = idpList.length > 1;
+
+        return (
+            <Message
+                data-componentid="jit-provisioning-mfa-in-sequence-warning-message"
+                data-testid="jit-provisioning-mfa-in-sequence-warning-message"
+                type="info"
+                // Semantic hides warning messages inside <form> by default.
+                // Overriding the behaviour here to make sure it renders properly.
+                content={ (
+                    <>
+                        {
+                            moreThan1IdP
+                                ? (
+                                    <>
+                                        Asgardeo requires the user&apos;s profile containing the
+                                        <i> mobile number</i> to configure 
+                                        <strong> SMS OTP</strong> with the following connections.
+                                        <ul className="mb-3">
+                                            { idpList?.map(({ name }:{name: string}, index: number) => (
+                                                <li key={ index }>
```

Comment:
Using index provided by `.map()` is an anti-pattern as it can cause hard-to-debug performance and rendering problems. 

Ref: https://robinpokorny.com/blog/index-as-a-key-is-an-anti-pattern/

```suggestion
                                                <li key={ name }>
```